examples/window/window_settings.rs

```diff
@@ -34,3 +36,18 @@ fn toggle_cursor(input: Res<Input<KeyCode>>, mut windows: ResMut<Windows>) {
         window.set_cursor_visibility(!window.cursor_visible());
     }
 }
+
+/// This system toggles DPI usage when enter is pressed
+fn toggle_dpi(input: Res<Input<KeyCode>>, mut windows: ResMut<Windows>) {
```

Comment:
🔥 
It would be quite good to have some text/ui which ignores the physical resolution in this example, or else add this code to a ui example.